Tasks/PythonScriptV0/task.json

```diff
@@ -0,0 +1,112 @@
+{
+    "id": "6392F95F-7E76-4A18-B3C7-7F078D2F7700",
+    "name": "PythonScript",
+    "friendlyName": "Python Script",
+    "description": "Run a Python script.",
+    "helpMarkDown": "",
+    "category": "Utility",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 135,
+        "Patch": 0
+    },
+    "preview": true,
+    "demands": [],
+    "instanceNameFormat": "Run Python script",
+    "groups": [
+        {
+            "name": "advanced",
+            "displayName": "Advanced",
+            "isExpanded": false
+        }
+    ],
+    "inputs": [
+        {
+            "name": "targetType",
+            "type": "radio",
+            "label": "Type",
+            "required": true,
+            "defaultValue": "filePath",
+            "helpMarkDown": "Target script type: File path or Inline",
+            "options": {
+                "filePath": "File path",
+                "inline": "Inline"
+            }
+        },
+        {
+            "name": "filePath",
```

Comment:
Can you rename this input from "filePath" to "scriptPath" so that the designer and YAML views match?